zookeeper-server/src/main/java/org/apache/zookeeper/server/ZooTrace.java

```diff
@@ -33,8 +33,6 @@
 
     public static final long CLIENT_REQUEST_TRACE_MASK = 1 << 1;
 
-    public static final long CLIENT_DATA_PACKET_TRACE_MASK = 1 << 2;
```

Comment:
it is better to keep this constant here.
we can add a comment and say that this is obsolete

the same applies to `JMX_TRACE_MASK`

Reply:
get